packages/websocket-client/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import * as WebSocket from 'isomorphic-ws';
+
+import { spawn, ensure, Resource } from '@effection/core'
+import { createQueue, Subscription } from '@effection/subscription';
+import { on, once } from '@effection/events';
+
+interface WebsocketClient<TIncoming = unknown, TOutgoing = TIncoming> extends Subscription<TIncoming> {
+  send(message: TOutgoing): void;
```

Comment:
Are there cases where having `send()`  be synchronous can cause the execution of illegal instructions?

Reply:
Not sure what you mean by this?